### DISCUSSION TITLE
Right Screen cut of on Ultrawide remote screen

### DISCUSSION THREAD

**Discussion by stilicrafter** [rustdesk/rustdesk#4775] [General]
### Bug Description

Hi,
when i view a remote ultra wide screen, the right side is cut off.

### How to Reproduce

Have a device like surface book 2 and view someone else screen that is ultrawide

### Expected Behavior

you see whole screen.
i cannot see right side like maimize etc.

### Operating system(s) on local side and remote side

Win 10 -> Win 10

### RustDesk Version(s) on local side and remote side

1.1.9 -> 1.1.9

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/22886797/698f6a6a-38e9-4eb1-b7c5-cc1999601519)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2988

**Comment by stilicrafter** [comment]
The ultrawide is an lg screen, but dont know exakt model
